Classes/PullRequestReviews/PullRequestReviewCommentsViewController.swift

```diff
@@ -29,7 +30,10 @@ final class PullRequestReviewCommentsViewController: MessageViewController,
         return f
     }()
 
-    init(model: IssueDetailsModel, client: GithubClient, autocomplete: IssueCommentAutocomplete) {
+    init(model: IssueDetailsModel,
```

Comment:
nit: if we're updating the style, can you put the first param on its own line? Consistent w/ the rest of the app.

```swift
init(
  model: IssueDetailsModel,
  client: GithubClient,
  // etc
```